src/main/java/duke/enums/Message.java

```diff
@@ -0,0 +1,68 @@
+package duke.enums;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * An Enumeration of the various Messages words that shall be used by Duke's UI
+ */
+public enum Message {
```

Comment:
I like how you created an enum class for the error messages. It would make all the errors more organised.